distribution/docs/src/main/resources/content/_managing/_installing/configure-single-sign-on.adoc

```diff
@@ -0,0 +1,76 @@
+:title: System Configuration Settings
+:type: installing
+:status: published
+:summary: Configure Single Sign On(SSO) page of installer.
+:project: ${branding}
+:order: 07
+
+.[[_configuring_sso]]Configure Single Sign On(SSO)
+****
+Configure Single Sign On method: SAML or OIDC.
+
+.SAML SSO
+Enter the URLs for the IdP metatdata and set other options.
+
+IdP Server::
+Configure ${BRANDING}'s' internal Identity Provider Server.
+
+. *_SP Metadata_*: The metatada of the desired Service Provider to connect to the internal IdP. Can be configured as an HTTPS URL (https://), file URL (file:), or an XML block(`&lt;md:EntityDescriptor&gt;...&lt;/md:EntityDescriptor&gt;`).
+
+. *_Require Signed AuthnRequests_*:
+Toggle whether or not to require signed AuthnRequests. When off, unsigned AuthnRequests will be rejected.
+
+. *_Limit RelayStates to 80 Bytes_*: Toggle whether or not to restrict the RelayState length to 80 bytes. When off, messages with RelayStates over 80 bytes will be rejected.
```

Comment:
❓ Should we include a comment here about why we might want to reject `RelayState`s of over 80 bytes in length?

Reply:
Yes. I'll track that down

Reply:
This is in section 3.4.3 of [Bindings for the OASIS Security Assertion Markup Language (SAML) V2.0 – Errata Composite](https://www.oasis-open.org/committees/download.php/56780/sstc-saml-bindings-errata-2.0-wd-06-diff.pdf) 

> RelayState data MAY be included with a SAML protocol message transmitted with this binding. The value MUST NOT exceed 80 bytes in length and SHOULD be integrity protected by the entity creating the message.

Reply:
updated